### PR TITLE
[PLAYER-5647] Fixed ad completed in Advanced Sample App

### PR DESCRIPTION
**Pull request by maxkupetskii**

### CODE REVIEW

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m [hunk 1]
```diff
@@ -131,7 +131,10 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
 
   switch ((OOOoyalaPlayerState)[change[NSKeyValueChangeNewKey] intValue]) {
     case OOOoyalaPlayerStateCompleted:
-      [self.notifier notifyAdCompleted];
+      [self.notifier notifyAdCompletedFromPlugin:OOAdProviderNone
+                             timeSinceImpression:0
+                                       isSkipped:NO
```
Comment (oSulyma) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m:136]: What if ad was skipped? This app and this class has appropriate event `- (void)skipAd {}`
Can Analytics be notified that ad is completed in way of skipped?

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m [hunk 2]
```diff
@@ -131,7 +131,10 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
 
   switch ((OOOoyalaPlayerState)[change[NSKeyValueChangeNewKey] intValue]) {
     case OOOoyalaPlayerStateCompleted:
-      [self.notifier notifyAdCompleted];
+      [self.notifier notifyAdCompletedFromPlugin:OOAdProviderNone
+                             timeSinceImpression:0
```
Comment (oSulyma) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m:135]: No need for measuring ad duration (timeSinceImpression:0)?

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m (hunk 1 above)
Comment (oSulyma) [reply AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m:136]: I launched app and noticed that there in no button 'skip ad' during playback of ad. So if it's as intended, my question above can be canceled.

**Review by oSulyma — COMMENTED** [review]
I opened two questions

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m (hunk 2 above)
Comment (maxkupetskii) [reply AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Utils/OOSamplePlugin.m:135]: Yes

**Review by oSulyma — APPROVED** [review]
LGTM

**Review by DmitriiMaslov — APPROVED** [review]

**Pull request merged**